docs/outlook/one-outlook.md

```diff
@@ -79,14 +79,20 @@ For guidance on how to sideload your add-in, see [Sideload Outlook add-ins for t
 
 ## Debug your add-in
 
-To debug an add-in installed in the new Outlook on Windows desktop client, first [sideload the add-in to Outlook on the web](sideload-outlook-add-ins-for-testing.md). Then, in a command prompt, run the following command to open the new Outlook on Windows client and an instance of the Microsoft Edge DevTools.
+To debug an add-in installed in the new Outlook on Windows desktop client, perform the following:
 
-```command&nbsp;line
-olk.exe --devtools
-```
+1. Close the new Outlook on Windows client if you already have it open.
+1. Then, in a command prompt, run the following command to open the new Outlook on Windows client and an instance of the Microsoft Edge DevTools.
 
-> [!TIP]
-> After running the command, the DevTools window stays open, so that you can debug an add-in's task pane as you open and close it. If you close the DevTools window, you must close your Outlook client first before running `olk.exe --devtools` again.
+  ```command&nbsp;line
+  olk.exe --devtools
+  ```
+
+  > [!TIP]
+  > After running the command, the DevTools window stays open, so that you can debug an add-in's task pane as you open and close it. If you close the DevTools window, you must close your Outlook client first before running `olk.exe --devtools` again.
```

Comment:
If the final 2 steps are meant to be steps 3 and 4, you likely need to indent these further

```suggestion
    ```command&nbsp;line
    olk.exe --devtools
    ```

    > [!TIP]
    > After running the command, the DevTools window stays open, so that you can debug an add-in's task pane as you open and close it. If you close the DevTools window, you must close your Outlook client first before running `olk.exe --devtools` again.
```

Reply:
Thanks for catching this!